reference/checkouts.v3.yml

```diff
@@ -1351,15 +1351,169 @@ definitions:
                       message:
                         description: Limited to 200 characters.
                         type: string
-                    required:
-                      - theme
-                      - amount
-                      - sender
-                      - recipient
-              required:
-                - physical_items
-                - digital_items
-                - gift_certificates
+                required:
+                - theme
+                - amount
+                - sender
+                - recipient
+                custom_items:
```

Comment:
Do all of these fields apply to `custom_items`?

Reply:
I fixed it. Custom_items should have 
* extended_list_price
* id
* image_url
* list_price
* name
* quantity
* sku

Reply:
@traci707  Can you please check on the custom_items array again? The code still shows other fields.

Reply:
Verified that `custom_items` looks correct in Stoplight.